.gitignore

```diff
@@ -41,4 +41,3 @@ target/
 *.crc
 demo/jars/*
 demo/notebook/.ipynb_checkpoints/*
-my_config.yaml
```

Comment:
Ignoring this file is intentional since people try to run the demo and create a `my_config.yaml` and try to check it in. Please leave this in the `.gitignore`

Reply:
The changes have been made and tested too. Workflow run in forked repo is also working fine. Please review and approve.
<img width="1071" alt="image" src="https://github.com/user-attachments/assets/f071c287-03a9-4a1e-908b-2898931bae44" />